bedrock/firefox/templates/firefox/campaign.html

```diff
@@ -0,0 +1,76 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% add_lang_files "firefox/campaign" %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{% block canonical_urls %}
+  {# the SEM campaign page should set canonical to the /firefox/new/ page. #}
+  <link rel="canonical" href="{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/">
+{% endblock %}
+
+{% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
+
+{%- block page_title -%}{{_('Free Web Browser')}}{%- endblock -%}
+
+{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!')}}{% endblock %}
```

Comment:
Nit: web is capitalized in "a free Web browser"

Reply:
These are all existing strings and heavily translated, so I'm not gonna change this one at this time. Good call though 👍